newton/tests/test_parent_force.py

```
@@ -0,0 +1,149 @@
+
+            newton.eval_fk(model, model.joint_q, model.joint_qd, state_0)
+            solver.step(state_0, state_1, None, None, dt)
+
+            parent_f = state_1.body_parent_f.numpy()[0]
+            weight = model.body_mass.numpy()[0] * 9.81
+
+            np.testing.assert_allclose(parent_f[:3], [0, 0, weight], rtol=1e-4)
+            np.testing.assert_allclose(parent_f[3:6], [0, 0, 0], atol=1e-2)
+
+
+def test_parent_force_centrifugal(test, device, solver_fn):
```

Ruff (0.14.14)

[warning] Unused function argument: `test`

(ARG001)